llvm/test/MC/AArch64/mve-sqrshr-uqrshl-earlyclobber.s

```diff
@@ -0,0 +1,6 @@
+@ RUN: not llvm-mc -triple armv8.1m.main -mattr=+mve < %s 2>&1 | FileCheck %s
```

Comment:
This test can be moved too :)

Reply:
Oh I just checked this and seemingly my eyes skipped right over it. Thank you!